### PR TITLE
Extend the length of AppId from 32 to 64 characters.

### LINKED ISSUE #3213: appId字段长度真的需要超过32个字符
apollo配置服务限制为32个字符是真的非常太不方便了，项目的id基于每个公司有自己的规则，加上公司名称和团队名称这些前缀，很容易就超过32个了，至于你们考虑的是否能记住，这个我认为并不需要考虑，记不住的人自然会使用短的，不应该怕人记不住就强制不允许使用长的名称啊，况且谁会记这个id呢，没有必要记住，看项目时可以看Name的。强烈建议放开appId字段长度!!!这非常影响大家的使用，我把数据库中的字段长度都改了，但前端校验还是会限制为32，暂时还不想改代码自己编译，请问可以加到下一个版本的功能中吗？
我看到有好几次讨论这个相关问题
#2418

### PR DESCRIPTION
**Pull request by Macrame**
## What's the purpose of this PR

在项目中经常会遇到公司的项目有固定的前缀和格式，项目ID大于32字符的情况，这次扩展为64字符。有其它人也遇到了这个问题，迫切需要扩展这个字段的长度限制。

## Which issue(s) this PR fixes:
Fixes #3213

## Brief changelog
1. 把表中AppID字段的长度从32改为64
2. 把前端javascript中校验规则中32改为64

Follow this checklist to help us incorporate your contribution quickly and easily:

- [x] Read the [Contributing Guide](https://github.com/ctripcorp/apollo/blob/master/CONTRIBUTING.md) before making this pull request.
- [x] Write a pull request description that is detailed enough to understand what the pull request does, how, and why.
- [x] Write necessary unit tests to verify the code.
- [x] Run `mvn clean test` to make sure this pull request doesn't break anything.

### CODE REVIEW

**Comment by nobodyiam** [comment]
Please update the ddl for [k8s](https://github.com/ctripcorp/apollo/tree/master/scripts/apollo-on-kubernetes/db) and [docker-quick-start](https://github.com/ctripcorp/apollo/tree/master/scripts/docker-quick-start/sql).
Also don't forget the delta ddl in [flyway](https://github.com/ctripcorp/apollo/tree/master/scripts/flyway) and [delta](https://github.com/ctripcorp/apollo/tree/master/scripts/sql/delta) folders

**Comment by Macrame** [comment]
I had committed again.

File: scripts/flyway/configdb/V1.0.0__initialization.sql [hunk 1]
```diff
@@ -46,7 +46,7 @@ DROP TABLE IF EXISTS `AppNamespace`;
 CREATE TABLE `AppNamespace` (
   `Id` int(10) unsigned NOT NULL AUTO_INCREMENT COMMENT '自增主键',
   `Name` varchar(32) NOT NULL DEFAULT '' COMMENT 'namespace名字，注意，需要全局唯一',
-  `AppId` varchar(32) NOT NULL DEFAULT '' COMMENT 'app id',
+  `AppId` varchar(64) NOT NULL DEFAULT '' COMMENT 'app id',
```
Comment (nobodyiam) [inline scripts/flyway/configdb/V1.0.0__initialization.sql:49]: To make flyway work, we need to create a new file with an increment version, such as `V1.2.0__appid_extension.sql`.

```sql
Use ApolloConfigDB;
alter table `AppNamespace`  change AppId AppId varchar(64) NOT NULL DEFAULT 'default' COMMENT 'app id';
alter table `Cluster`  change AppId AppId varchar(64) NOT NULL DEFAULT 'default' COMMENT 'app id';
alter table `GrayReleaseRule`  change AppId AppId varchar(64) NOT NULL DEFAULT 'default' COMMENT 'app id';
alter table `Instance`  change AppId AppId varchar(64) NOT NULL DEFAULT 'default' COMMENT 'app id';
alter table `InstanceConfig` change ConfigAppId ConfigAppId varchar(64) NOT NULL DEFAULT 'default' COMMENT 'Config App Id';
alter table `ReleaseHistory`  change AppId AppId varchar(64) NOT NULL DEFAULT 'default' COMMENT 'app id';
```

File: scripts/flyway/portaldb/V1.0.0__initialization.sql [hunk 2]
```diff
@@ -46,7 +46,7 @@ DROP TABLE IF EXISTS `AppNamespace`;
 CREATE TABLE `AppNamespace` (
   `Id` int(10) unsigned NOT NULL AUTO_INCREMENT COMMENT '自增主键',
   `Name` varchar(32) NOT NULL DEFAULT '' COMMENT 'namespace名字，注意，需要全局唯一',
-  `AppId` varchar(32) NOT NULL DEFAULT '' COMMENT 'app id',
```
Comment (nobodyiam) [inline scripts/flyway/portaldb/V1.0.0__initialization.sql:49]: To make flyway work, we need to create a new file with an increment version, such as `V1.1.0__appid_extension.sql`.

```sql
Use ApolloPortalDB;

alter table `AppNamespace`  change AppId AppId varchar(64) NOT NULL DEFAULT 'default' COMMENT 'app id';
```

File: scripts/flyway/configdb/V1.0.0__initialization.sql (hunk 1 above)
Comment (Macrame) [reply scripts/flyway/configdb/V1.0.0__initialization.sql:49]: Yes, I forgot it. I'll commit later.

**Pull request merged**